swap/log.go

```diff
@@ -0,0 +1,83 @@
+package swap
+
+import (
+	log "github.com/ethereum/go-ethereum/log"
+)
+
+const (
+	// CallDepth is set to 1 in order to influence to reported line number of
+	// the log message with 1 skipped stack frame of calling l.Output()
+	CallDepth = 1
```

Comment:
Where is this variable used? It looks like it is not used anywhere

Reply:
Removed, this was used before, when it was part of the log package.
Not used anymore, good point.